metrics-jmx/src/main/java/com/codahale/metrics/jmx/JmxReporter.java

```diff
@@ -622,6 +625,7 @@ public void onMeterAdded(String name, Meter meter) {
             } catch (JMException e) {
                 LOGGER.warn("Unable to register meter", e);
             }
+            return null;
```

Comment:
Why don't we return the `Meter` which has been passed into the method when it was added?